### ISSUE TITLE
Server Errors Regarding Extra Origins

### ISSUE THREAD

**Issue by PhantomsKeep** [MoriyaShiine/extra-origins#121]
Hello! I'm currently running a MC server using Origins, Extra Origins, Mob Origins, and Origin Classes. I loaded in just fine to the server and was able to choose an origin and class with no problems. When other people joined they immediately got kicked out of the server. At first I thought it was a network error; they restarted their routers, no dice. We changed the server's location, they still couldn't get in. We took apart the pack and figured out that Extra Origins was causing the issue. There was no crash log or warning. I saw other people were having the same issue when I tried to find a solution, but I couldn't find where they fixed it for the 1.17-1 version of the mod.

Here's all the mod versions we are using:
extraorigins-1.17-1
fabric-api-0.35.2+1.17
moborigins-1.5.0
Origins-1.17-1.0.0
Origins-Classes-1.17-1.2.1
Pehkui-2.2.1+1.14.4-1.17

The server host (if it helps any) is Apex Hosting.

While looking at the Curseforge page, I saw that the mod was updated. We also tried the 1.17-2 version and it still did not work. Does Extra Origins clash with the code of any of the mods above? I'm really sorry I don't have a crash log to show you. I've been trying to fix this for about 4 hours now and I still couldn't find a solution.

**Comment by MoriyaShiine** [comment]
I'm also able to join just fine using the same mod list. Can you send the server's latest log?

**Comment by PhantomsKeep** [comment]
Sorry for the late response, I was at work! Here's the error message I kept seeing (I don't have the full console history at the moment, I'm going to have to wait for another player to wake up and get online for that. The error message is the same, though.)

[Server] Server thread/WARN Player class_3222['Elfghoul'/97026, l='ServerLevel[TAKE4]', x=-705.69, y=74.49, z=1188.31] could not be synced because server networking isn't set up yet.
[Server] Server thread/INFO Elfghoul joined the game
[Disconnect] User Elfghoul has disconnected, reason: Disconnected
[Server] Server thread/INFO Elfghoul left the game

I hope this is a bit helpful for now? One person was apparently able to join the server after I took the EO mod out of the actual server, but when I put it back into the server he was still able to log in and play.
